file-formats/aifi/type/zif_aff_aifi_v1.intf.abap

```diff
@@ -139,72 +144,78 @@ INTERFACE zif_aff_aifi_v1
     BEGIN OF ty_general_information,
       "! <p class="shorttext">Integration Type</p>
       "! Integration type
-      integration_type              TYPE ty_integration_type,
+      integration_type             TYPE c LENGTH 4,
+      "! <p class="shorttext">Integration Type</p>
```

Comment:
Is this a `Integration Type Description`? Or is there a reason for leaving out `Description`? 

Same for `scenario_description`.

Reply:
Yes. The description field was added, in order to display the text instead of the value (we don´t want an enumeration). But the field label should remain "Integration Type".

Reply:
Same for the Scenario.